pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -196,7 +198,13 @@ func (p *PollerWorker) Poll(execAgentCommands agentCommandsExecutor, closeChan c
 	// do the first round to fill the cache immediately
 	execAgentCommands(p.AgentCommands)
 
-	ticker := time.NewTicker(time.Second * p.CallTick)
+	pollStableInterval := p.CallTick * time.Second
+	pollInterval := pollStableInterval
+
+	if pollInitialInterval < pollStableInterval {
+		pollInterval = pollInitialInterval
+	}
```

Comment:
It cost me a bit to understand this initialization code; quite probably, naming the variable `maxPollInterval` instead of `pollStableInterval` would have made it easier.